app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details_edit.scala.html

```diff
@@ -0,0 +1,150 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import helper._
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.RulingController
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController
+@import uk.gov.hmrc.tariffclassificationfrontend.forms.DecisionFormData
+@import uk.gov.hmrc.tariffclassificationfrontend.models.response.ScanStatus
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case, StoredAttachment}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.{input_text, input_textarea}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{attachments_list, tab_heading}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.error_summary
+@(c: Case, files: Seq[StoredAttachment], decisionForm : Form[DecisionFormData], startAtTabIndex : Option[Int] = None)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None, customTitle = Some(messages("page.title.edit.ruling"))) {
```

Comment:
I've pulled the form out of the tab view to be consistent with other forms. Verified this is fine with Aimee, it was on the backlog to do anyway